Library/Formula/xorg-util-macros.rb

```diff
@@ -0,0 +1,19 @@
+class XorgUtilMacros < Formula
+  desc "Miscellaneous utility macros required by the X.org codebase."
+  homepage "http://www.x.org"
+  url "http://xorg.freedesktop.org/archive/individual/util/util-macros-#{version}.tar.gz"
+  version "1.19.0"
+  sha256 "0d4df51b29023daf2f63aebf3ebc638ea88efedfd560ab5866741ab3f92acaa1"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "true"
```

Comment:
I got your point but I know nothing about those macros and I don't plan on learning how to use them. I just need them to build a few other tools. This formula has 15 lines and the code in the tarball about the same. You already wrote more than 15 lines of comment even if I followed exactly all the guidelines.

My goal is to build melt and kdenlive (that depend on this) and contribute them as well. If too much trouble I will just keep them for me.